src/test/java/com/teragrep/cfe_39/PruneTest.java

```diff
@@ -0,0 +1,130 @@
+package com.teragrep.cfe_39;
```

Comment:
This whole file is a questionmark for me

Reply:
That file is actually going to be removed as it was only used for testing different kinds of pruning approaches.